### PR TITLE
Update create_tables.sql

### PR DESCRIPTION
**Pull request by iwle**
Update relation creation according to the latest ER diagram.

### CODE REVIEW

File: database/create_tables.sql [hunk 1]
```diff
@@ -27,9 +26,11 @@ CREATE TABLE Likes (
 );
 
 CREATE TABLE Backs (
-    backs_id serial PRIMARY KEY,
     user_id int REFERENCES Users(user_id),
-    project_id int REFERENCES Projects(project_id)
+    project_id int REFERENCES Projects(project_id),
+    transaction_id int REFERENCES Transactions(transaction_id),
+        back_date datetime NOT NULL,
```
Comment (ptvrajsk) [inline database/create_tables.sql:32]: Err this one the indentation off?

File: database/create_tables.sql [hunk 2]
```diff
@@ -39,35 +40,31 @@ CREATE TABLE Transactions (
...
-    project_id int REFERENCES Projects(project_id),
+    project_name int REFERENCES Projects(project_name),
     user_id int REFERENCES Users(user_id),
+    create_date datetime NOT NULL,
     CONSTRAINT creates_constraint PRIMARY KEY(user_id, project_id)
 );
 
 CREATE TABLE SearchHistory (
-    search_id serial PRIMARY KEY,
+    search_timestamp timestamp,
```
Comment (ptvrajsk) [inline database/create_tables.sql:50]: Actually there's an issue here? 
If 2 users search something at the same time then the timestamp will be the same and thus cannot store both searches with different text and same timestamp?

File: database/create_tables.sql [hunk 3]
```diff
@@ -39,35 +40,31 @@ CREATE TABLE Transactions (
...
-    cv_number varchar(3),
-
-    CONSTRAINT
-      credit_card_expiry_month
-      CHECK (credit_card_expiry_month BETWEEN 1 AND 12),
-    CONSTRAINT
-      credit_card_expiry_year
-      CHECK (credit_card_expiry_year > (extract(epoch from date_trunc('year', CURRENT_DATE))::int))
+    amount numeric,
+    CONSTRAINT wallets_constraint PRIMARY KEY(user_id, amount)
```
Comment (ptvrajsk) [inline database/create_tables.sql]: Can the primary key be just the Foreign key? Hahah cause the amount feels abit redundant.

**Review by ptvrajsk — CHANGES_REQUESTED** [review]
Some minor queries.

File: database/create_tables.sql (hunk 1 above)
Comment (iwle) [reply database/create_tables.sql:32]: Yeah the indentation is off. Thanks!

File: database/create_tables.sql (hunk 2 above)
Comment (iwle) [reply database/create_tables.sql:50]: Oh, wait. I forgot to add in the `user_id` in the primary key. Thanks for checking!

File: database/create_tables.sql (hunk 3 above)
Comment (iwle) [reply database/create_tables.sql]: But then what will the primary key for `Wallets` be? Unless we make `Wallets` a weak entity set.

**Review by ptvrajsk — APPROVED** [review]
Skimmed through, seems okay 👍

**Pull request merged**